libmscore/types.h

```diff
@@ -247,6 +247,33 @@ enum class AccidentalType : char {
       ///\}
       };
 
+//---------------------------------------------------------
+//   NoteType
+//---------------------------------------------------------
+
+enum class NoteType {
+      ///.\{
+      NORMAL        = 0,
+      ACCIACCATURA  = 0x1,
+      APPOGGIATURA  = 0x2,       // grace notes
+      GRACE4        = 0x4,
+      GRACE16       = 0x8,
+      GRACE32       = 0x10,
+      GRACE8_AFTER  = 0x20,
+      GRACE16_AFTER = 0x40,
+      GRACE32_AFTER = 0x80,
+      INVALID       = 0xFF
+      ///\}
+};
```

Comment:
Still 6 space indent missing

Reply:
Please be more specific as I must be missing your issue.

Even looking at your post and the email I received I count out exactly 6 preceding spaces for each interior line in the NoteType definition.

Are we talking apples and oranges here?

-Dale